progFun-20162-teoria

joagranadosme opened the pull request:
Imagenes
Ejemplos Fibonacci
Modularidad
Transparencia referencial
Polimorfismo paramétrico
Funciones de primer orden y orden superior

joagranadosme committed:
Add files via upload

Imagenes
Ejemplos Fibonacci
Modularidad
Transparencia referencial
Polimorfismo paramétrico
Funciones de primer orden y orden superior

amrondonp commented:
Saludos.

Compañeros, el aporte que le hacen a la pagina es muy básico, veo que hablan de Modularidad y Transparencia referencial pero no hay ejemplos que ilustren. Para Polimorfismo paramétrico, hacen falta más ejemplos y decir que implicaciones tienen, en que casos podría usarse.

En la parte de funciones de orden superior los ejemplos también son básicos, por favor incluir más ejemplos

El ejemplo de fibonacci esta bien, está completo.

Y por ultimo, por favor revisar la ortografía del documento.

joagranadosme committed:
progFun-20162-teoria

Correcciones:
Modularidad
Transparencia referencial
Polimorfismo paramétrico.
Ejemplo de funciones de primer orden y orden superior.
